README.md

```diff
@@ -76,13 +76,14 @@ Please refer to the almond-tokenizer documentation for details.
 To generate a synthetic set, use:
 
 ```
-genie generate --language en --template template.genie --thingpedia thingpedia.json --dataset dataset.tt -o synthetic.tsv
+genie generate --locale en --template template.genie --thingpedia thingpedia.json --dataset dataset.tt -o synthetic.tsv
 ```
 
 The `--template` flag can be used to point to a template file definining the construct templates,
-in Genie language. Multiple `--template` flags
-can be used to load multiple template files. If Genie has write permissions on the directory
-containing the template files, the files will be automatically compiled to JS and cached.
+in Genie language. E.g. `languages/en/thingtalk.genie` is the one for English sentence synthesis.
+Multiple `--template` flags can be used to load multiple template files.
+If Genie has write permissions on the directory containing the template files,
+the files will be automatically compiled to JS and cached.
```

Comment:
We should probably say that we don't cache any more...

Reply:
I'll remove the last line "If Genie ... and cached."